win_build/boinc_finalize.targets

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<Target Name="AfterBuild">
+		<ItemGroup>
+			<LOCALE Include="..\locale\**\*.*" />
+			<CERT Include="..\curl\ca-bundle.crt" />
+		</ItemGroup>
+		<Copy SourceFiles="@(LOCALE)" DestinationFolder="$(OutDir)\locale\%(RecursiveDir)" SkipUnchangedFiles="true" />
+		<Copy SourceFiles="@(CERT)"   DestinationFolder="$(OutDir)"                        SkipUnchangedFiles="true" />
+	</Target>
+</Project>
```

Comment:
Please add new line

Reply:
I will. All other projects and target doesn't have new line.

Reply:
I know but anyway for new files it's better to add it :)